test/config.spec.ts

```diff
@@ -204,3 +204,13 @@ describe('Config', () => {
     expect(() => new Config({ nullYear: 101 })).toThrowError('Config parameter nullYear should be at most 100')
   })
 })
+
+describe('getConfig', () => {
+  it('should copy returned values', () => {
```

Comment:
Can we also check that it is not an instance of `Config` just to be sure? The main requirement was to have a literal object returned